.github/workflows/master.yml

```diff
@@ -102,15 +102,15 @@ jobs:
             pyarrow-version: 0.14.1
           - python-version: 3.7
             spark-version: 2.4.6
-            pandas-version: 1.0.5
+            pandas-version: 1.1.0
```

Comment:
As some of functions changed the behavior, shall we keep using `1.0.5` here?

Reply:
Thanks, @ueshin .
Seems like test failure since the behavior has been changed.
I'll open the PR to fix them.